### ISSUE TITLE
Plugin is not visible on the admin dashboard

### ISSUE THREAD

**Issue by dgokcin** [colibris-xyz/strapi-plugin-site-publisher#66]
Helloo

I have installed the plugin to strapi v 4.4.5 with yarn and I can see it listed under http://localhost:1337/admin/list-plugins. However, I can not see it on the admin panel. I also see in the logs that it is enabled.

```
module.exports = ({ env }) => ({
  upload: {
    enabled: true,
  },
  "import-export-entries": {
    enabled: true,
  },
  'site-publisher': {
    config: {
      owner: 'username', // The Github organisation or user
      repo: 'reponame', // The name of the repository
      workflow_id: 'rebuild.yml', // The workflow_id or filename
      token: env('GITHUB_TOKEN'), // The GitHub personal access token with access to trigger workflows and view build status
      branch: 'master', // The branch the workflow should be triggered on
      inputs: {
        // Optional inputs to pass through to the GitHub workflow
        some_input: 'Some value',
        some_other_input: 'Some other value',
      }
    }
  },
});


```

Could you please help me out?

**Comment by tms0** [comment]
Hi,

All I can see is that you use the [website builder](https://market.strapi.io/plugins/strapi-plugin-website-builder) plugin, this project is about the [site publisher](https://market.strapi.io/plugins/strapi-plugin-site-publisher) plugin.

**Comment by dgokcin** [comment]
@tms0 sorry for the confusion. I experimented with so many plugins that I mixed the repositories. I updated the first comment of mine with the correct plugins config and attaching an ss from the list-plugins page. Any idea why I can not see the site publisher plugin?

<img width="1323" alt="image" src="https://user-images.githubusercontent.com/33603535/201462908-06f5cb47-b803-43ff-9857-5e33e49e8910.png">
